.pre-commit-config.yaml

```diff
@@ -0,0 +1,61 @@
+repos:
+- repo: local
+  hooks:
+  - id: flake8
+    name: flake8
+    entry: flake8
+    language: system
+    types: [python]
+  - id: pylint
+    name: pylint
+    entry: pylint typedspark
+    language: system
+    types: [python]
+    args:
+      [
+        '-rn', # Only display messages
+        '-sn', # Don't display the score
+      ]
+  - id: mypy
+    name: mypy
+    entry: mypy
+    language: system
+    types: [python]
+  - id: pyright
+    name: pyright
+    entry: pyright
+    language: system
+    types: [python]
+  - id: black
+    name: black
+    entry: black --check
+    language: system
+    types: [python]
+  - id: isort
+    name: isort
+    entry: isort --check
+    language: system
+    types: [python]
+  - id: docformatter
+    name: docformatter
+    entry: docformatter --black -c
+    language: system
+    types: [python]
+  - id: pytest-no-spark
+    name: pytest-no-spark
+    entry: python -m pytest tests/_core/test_column.py::test_column_reference_without_spark_session
```

Comment:
Can we do this with a [pytest mark](https://docs.pytest.org/en/latest/how-to/mark.html) of the test instead of hardcoding it into the pre-commit-config?

Reply:
Oh, nice! 

Didn't know that, fixed it.